libraries/AP_ExternalAHRS/AP_ExternalAHRS_MicroStrain7.h

```diff
@@ -50,6 +50,13 @@ class AP_ExternalAHRS_MicroStrain7: public AP_ExternalAHRS_backend, public AP_Mi
         build_packet();
     };
 
+protected:
+
+    uint8_t num_gps_sensors(void) const override
```

Comment:
nitpick; normally the curly bracket is on the same line as the definition in .h files.  Not a blocker of course.  Personally I'd put this all on one line.